content/departments/cloud/technical-docs/v2.0/onprem_data_migration.md

```diff
@@ -216,18 +199,39 @@ Audit logs are generated for bucket access in the project's logs, under log entr
 
 ## Execute data migration
 
-### Ensure GCP permission
+### Disable alerting and scale down cloud instance
 
-Visit go/cloud-ops and locate the instance, then request access to Cloud infra via the provided Entitle link.
+Once the database backups and instance summary have been uploaded by the customer, the first step before processing the migration is to disable the instance alerting and scale down the Cloud instance.
 
-### Reset databases
+1. Update mi2 to the latest version:
 
-First, prepare the Cloud database for import. Make sure all Sourcegraph pods are scaled down in the cloud instance with the exception of `cloud-sql-proxy`
+```
+sg cloud install
+```
+
+2. Extract the Cloud instance from the control plane, following the instructions in the _"Extract instance from control plane (break glass)"_ section in the instance-specific dashboard from [go/cloud-ops](https://cloud-ops.sgdev.org/dashboard/environments/prod)
+
+3. Disable alerting by editing the instance `config.yaml` in [`sourcegraph/cloud`](https://github.com/sourcegraph/cloud) as follows. Make sure to deploy the terraform changes as well.
+
+```diff
+spec:
+  debug:
+-    enableAlerting: true
++    enableAlerting: false
+```
+
+4. Commit and submit your changes as a pull request. After merging and confirming the apply in Terraform Cloud, proceed with scaling down the instance:
```

Comment:
we can apply locally with `mi2 instance tfc deploy -e prod -s navan -auto-approve -force-ignore-stack-dependencies -target monitorin`

Reply:
good call out, added